react-app/src/components/common/Tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useState, Fragment } from "react";
+import cn from "classnames";
+import { Transition } from "@headlessui/react";
+
+const Tooltip: React.FC<{
+  children: React.ReactNode;
+  title?: string;
+  content?: React.ReactNode;
```

Comment:
can this be `content?: React.ReactNode | string` since they are rendered exclusively?